lang/translation.json

```diff
@@ -80,9 +71,18 @@
 	      "en": "Links",
 	      "nl": "Links"
 	  },
+	  {
+	    "tag" : "word_maximum_value",
+	    "en" : "Filter on the maximal value",
+	    "nl" : "Filteren op maximale waarde"},
+	    {
+	      "tag" : "tool_maximum_value_1_expl",
+	      "en" : "",
+	      "nl": "Elke meetwaarde die de sensor doorgeeft kan informatie geven over de luchtkwaliteit van die plek. Echter, wanneer de sensor vies of beschadigd is, is de meetwaarde niet representatief voor de luchtkwaliteit. Vaak zijn dat dan hoge waardes van bijvoorbeeld 300. Voor de analyse wil je graag de luchtkwaliteit bekijken, zonder dat deze verstoord wordt door deze hoge waardes. Daarvoor kun je een filter zetten die de meetwaardes boven de maximale waarde niet tonen. Standaard staat de maximale waarde op het 10% percentiel van de sensormetingen. Je kunt ook zelf het filter instellen, zo kun je ook de hoge waardes van een sensor bekijken. Zet de maximale dan bijvoorbeeld op 1000 en bekijk de sensoren één voor één."
```

Comment:
'10% percentiel' moet waarschijnlijk 90-precentiel zijn (en/of ' op de waarde waaronder 90% van de metingen vallen')

Reply:
Thanks, I changed it to 90%